### ISSUE TITLE
Fix linting

### ISSUE THREAD

**Issue by seabbs** [epinowcast/epidist#101]
Linting checks are currently failing and the mix of old and new issues makes the CI broadly useless for review support (for example see #95). We either need to address all linting issues urgently or commit to dealing with those flagged by the lint in a given PR so that it passes. 

If the latter I will make linting required in PRs (hence it will block) but if the former we need a PR to address this (and then I will mak linting required).

**Comment by athowes** [comment]
Yeah agree, I think my favoured course of action would be to create a PR to fix all linting issues ASAP. (And then require lint passing.) This situation where making changes brings up lint issues that the changes didn't create is annoying.

Is there a way I can run the lint checker locally? It's just something like `lintr::lint` I imagine.

**Comment by seabbs** [comment]
yes its `lintr::lint_package()` I think
